src/main/webapp/2Dscript.js

```diff
@@ -0,0 +1,86 @@
+// These are gobal variables.
+let map;
+let pose;
+let runId;
+let dataType;
+
+fetchData();
+/**
+ * This function fetchs pose data from the RunInfo servlet,
```

Comment:
Omit "This function ".

"Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied 'This method ...' before it."

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments